src/schematics/ng-add/index.ts

```diff
@@ -54,14 +55,13 @@ ${detailsContent}
 `)
     }
   }
-}
-
```

Comment:
Is this line removal inteded? :see_no_evil:

Reply:
It's not a removal. the method is moved into the scope of the default method.